components/ActionSheet/src/private/MDCActionSheetHeaderView.h

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface MDCActionSheetHeaderView : UIView
+
+- (nonnull instancetype)initWithTitle:(nullable NSString *)title;
+
+- (nonnull instancetype)initWithTitle:(nullable NSString *)title message:(nullable NSString *)message;
+
+/** Header must be created with either headerWithTitle: */
+- (nonnull instancetype)initWithFrame:(CGRect)frame;
+
+/** Header must be created with either headerWithTitle: */
+- (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_UNAVAILABLE;
+
+@property (nonatomic, nullable, strong) NSString *title;
+
+@property (nonatomic, nullable, strong) NSString *message;
+
+@property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
```

Comment:
You can remove the readwrite as redundant unless we have another definition of this property that is readonly.

Reply:
Done